.github/workflows/pr-test.yml

```diff
@@ -61,12 +61,12 @@ jobs:
         if: startsWith(matrix.os, 'ubuntu')
         run: |
           sudo apt-get update
-          sudo apt-get install wget curl
+          sudo apt install -y wget apt-transport-https
```

Comment:
Should we switch all `apt-get` commands with `apt` ones, then?

Reply:
Ughh, my bad. Let's stick to `apt-get` all round since `apt` isn't meant for scripting/programmatic usage.